service/controller/v5/arm_templates/virtual_network_setup.json

```diff
@@ -10,7 +10,7 @@
     },
     "virtualNetworksAPIVersion":{
       "type":"string",
-      "defaultValue":"2016-09-01",
+      "defaultValue":"2016-12-01",
```

Comment:
Why does this change now? Does that have any implications for the client libs we use in the SDK? They are versioned in a similar way as well.

Reply:
AFAIK this doesn't have to be in sync with SDK. Different API versions can deal with the same resources.

Reply:
It was changed because I was going to use service endpoint for Storage. And old version of API didn't support it. But as doesn't work with azure-operator, I switched that back